Markdown/syntax_test_markdown.fenced_code_json.md

```diff
@@ -0,0 +1,30 @@
+//  SYNTAX TEST "Packages/Markdown/Markdown.sublime-syntax"
```

Comment:
Perl and Markdown prefer to keep syntax tests in one file. This should be respected by a foreign syntax.

Reply:
ok, sure.